src/logic/hooks/useEstimateTransactionGas.tsx

```diff
@@ -138,6 +138,21 @@ export const useEstimateTransactionGas = ({
       )
       const isOffChainSignature = checkIfOffChainSignatureIsPossible(isExecution, smartContractWallet, safeVersion)
 
+      if (!isCreation && !isExecution && !approvalAndExecution) {
+        setGasEstimation({
+          txEstimationExecutionStatus: EstimationStatus.SUCCESS,
+          gasEstimation: 0,
+          gasCost: '0',
+          gasCostFormatted: '< 0.001',
+          gasPrice: '1',
+          gasPriceFormatted: '1',
+          gasLimit: '0',
```

Comment:
Do you think we could move this object into `ethTransactions.ts` as a `getDefaultGas()` function?